util/task.py

```diff
@@ -56,7 +59,8 @@ def fully_submitted(user_id, year_id=None):
     }
 
 
-def any_submitted(user_id, year_id):
+def any_submitted(user_id: int, year_id: int)\
+        -> Tuple[model.Task, float, model.Wave, model.Prerequisite]:
```

Comment:
Nevypadlo tady `List`?
```suggestion
        -> List[Tuple[model.Task, float, model.Wave, model.Prerequisite]]:
```